src/lib/logger.ts

```diff
@@ -226,11 +240,43 @@ export class Logger {
       documentProperties: document.properties?.toJSON() || {},
       documentVisibility: document.visibility,
       documentChanges: document.changeCount,
-      commentText,
+      ...params,
       ...teacherNetworkInfo
     };
     Logger.log(event, parameters);
+  }
 
+  public static logCommentEvent({ focusDocumentId: focusDocumentId, focusTileId, isFirst, commentText, isAddingAComment }: ILogComment) {
```

Comment:
Can be just:
```
public static logCommentEvent({ focusDocumentId, focusTileId, isFirst, commentText, isAddingAComment }: ILogComment) {
```

Reply:
Might still exceed line length.  Might need to break onto two lines.

Reply:
If we're shaving characters to fit line length, we could replace `isAddingAComment` with `isAdding`, since the rest is rather implied by the fact that this is the `logCommentEvent()` function. 😉

Reply:
Changing to: isAdding